src/app/components/playback/playback_control.rs

```diff
@@ -149,6 +168,23 @@ impl PlaybackControl {
             .expect("error updating icon");
     }
 
+    fn update_repeat(&self, mode: &RepeatMode) {
+        let playback_image = match mode {
+            RepeatMode::Song => "media-playlist-repeat-song-symbolic.symbolic",
+            RepeatMode::Playlist => "media-playlist-repeat-symbolic",
+            RepeatMode::None => "media-playlist-consecutive-symbolic.symbolic",
```

Comment:
again no big deal but same as above, the .symbolic is not needed, "media-playlist-consecutive-symbolic" should work doesnt it?

Reply:
The extra symbolic for whatever reason makes the icon proper for me and some other themes. I think it is fine to leave it

Reply:
alright, it works after all :)